### DISCUSSION TITLE
Extract text propeties

### DISCUSSION THREAD

**Discussion by meghanaviyyapu** [pymupdf/PyMuPDF#1702] [Looking for help]
Is it possible to extract text matrix, character spacing, word spacing, text rise, text rendering mode, text knockout and text leading values with PyMuPDF?

**Comment by JorjMcKie** [comment]
Not with the high level interface. You must parse the contents stream yourself.

**Reply by meghanaviyyapu** [reply]
Is that approach feasible?

**Reply by JorjMcKie** [reply]
Depends on you 😉
For sure a lot of work.

**Reply by meghanaviyyapu** [reply]
Ok Thanks
